backend-base/src/test/resources/mock-response.json

```diff
@@ -0,0 +1,293 @@
+{
```

Comment:
Just to make sure. This is a valid Che 7.x API response?

Reply:
Yeah, this was grabbed from a running Che workspace